utk-cs340-fall24/SolSync
# feat: Send daily emails using data from firestore

# Description
Send daily emails to all users using data from Firestore.

# How to Test
1. Ask Kaylee to verify your email in Mailgun
2. Copy new env into project root
3. Use node version 20: `nvm use 20`
4. Navigate to daily email folder `cd lambdas/sendDailyEmails`
5. Install packages `npm i` (You must use npm)
6. Go back to project root `cd ../..`
7. Create an account and a habit in the app `pnpm start`
8. Send you emails `pnpm run daily-email` (This may take some time to complete as you will be sending a email to every user with a habit)

## Relevant Issues
#82
